### PR TITLE
introduction to Vim

### PR DESCRIPTION
**Pull request by is-ahmed**

### CODE REVIEW

**Comment by shehzaad777** [comment]
Looks good so far, we should edit the additional resources we use before merging though

**Review by is-ahmed — COMMENTED** [review]
Perhaps we could add a few images to help make things more clear

**Comment by shehzaad777** [comment]
> Perhaps we could add a few images to help make things more clear

Sounds good, I'll add an image for the modes for the moment

**Review by is-ahmed — COMMENTED** [review]
There seems to be a problem with the diagram, I can't see all of the text because the background is transparent

**Comment by shehzaad777** [comment]
Looks good, let me fix the diagram issue and I'm good to merge, check over it after too

**Review by is-ahmed — COMMENTED** [review]
Looks good thanks!

**Review by shehzaad777 — COMMENTED** [review]
looks good to me we can merge

File: Topics/Software_Tools/vim/vim.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+- Download the latest version for Windows
+
+- Run the installer and follow the installation prompts
+
+
+### Modes
+Here is a diagram of the modes available on vim and how to access them, they will be used in the following sections.
+![Image](./vim_modes.png)
+
```
Comment (sm86) [inline Topics/Software_Tools/vim/vim.md:94]: missing reference to image -> please specify that this image is taken from the below link

**Pull request merged**